### PR TITLE
Task 1 完成

### PR DESCRIPTION
**Pull request by frezcirno**

### CODE REVIEW

File: tzx/task 1/main.cpp [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+};
+
+bool isLedLight(const MyRotatedRect &rect, const Mat &img)
+{
+    if (rect.angleT() < 45 || rect.angleT() > 135)
+    {
+        return false;
+    }
+
+    if (rect.area() < 800.0 * img.size[0] * img.size[1] / 4497 / 3000)
```
Comment (ZhengQiushi) [inline tzx/task 1/main.cpp:55]: do not use magic number.

File: tzx/task 1/main.cpp [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    }
+
+    if (rect.aspectRatio() < 3)
+    {
+        return false;
+    }
+    return true;
+}
+
+void drawRotatedRect(Mat img, const RotatedRect &rect, const Scalar &color, int thickness = 1)
```
Comment (ZhengQiushi) [inline tzx/task 1/main.cpp]: you can add this function to your class **`class MyRotatedRect`**

File: tzx/task 1/main.cpp [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+        {
+            continue;
+        }
+        contourRects.emplace_back(rect);
+    }
+
+    return contourRects;
+}
+
+Scalar hsvToOpencvHsv(double h, double s, double v)
```
Comment (ZhengQiushi) [inline tzx/task 1/main.cpp:89]: please add some explanation about this function.

Comment (ZhengQiushi) [inline tzx/task 1/main.cpp:89]: please add some explanation about this function.

**Review by ZhengQiushi — COMMENTED** [review]
nice coding behavior! well done.

**Comment by frezcirno** [comment]
郑大佬，yyds！

**Comment by ZhengQiushi** [comment]
> 郑大佬，yyds！

please stop adding invalid comments and keep working...

File: tzx/task 1/main.cpp (hunk 3 above)
Comment (frezcirno) [reply tzx/task 1/main.cpp:89]: 从我用的取色器显示的hsv数值到OpenCV使用的hsv数值的转换函数（手动滑稽）
后面会去掉的

File: tzx/task 1/main.cpp (hunk 2 above)
Comment (frezcirno) [reply tzx/task 1/main.cpp]: 好的，下个版本改

**Pull request closed without merging**